### ISSUE TITLE
Button: partially disabled does not show focus indicator (IE/Edge)

### ISSUE THREAD

**Issue by hramakumar** [eBay/ebayui-core#715]
# Bug Report

## eBayUI Version: 2.6.0-2 and skin 7.3.2
partially disabled looks and behaves same as disabled
after button click
![image](https://user-images.githubusercontent.com/13157115/59713466-607e0580-91c4-11e9-950d-4f741ba48095.png)

expected: border focus
![image](https://user-images.githubusercontent.com/13157115/59713642-c10d4280-91c4-11e9-9cc0-b7f83652e6cf.png)

**Comment by ianmcburnie** [comment]
I'm a little confused by the desctription. Let me see if I have this right...

This different rendering engines behave differently for buttons when clicked with a mouse. IE, Edge and Chrome all show the focus indicator. Firefox and Safari do not. So I think we can close this as wontfix.

**Comment by hramakumar** [comment]
The issue is IE edge does not show focus indicator.
The actual behaviour is the first screenshot and the expected is the later.
I expect a focus indicator in IE edge, which I dont see.

**Comment by ianmcburnie** [comment]
I'm unable to repro in Edge, see focus outline below:

<img width="469" alt="Screen Shot 2019-10-02 at 8 48 53 AM" src="https://user-images.githubusercontent.com/38065/66059863-9637c780-e4f1-11e9-9736-a50b09275304.png">

For IE11 we can close as wont fix according to browser policy.
